lib/utils/note-utils.ts

```diff
@@ -42,9 +44,37 @@ export const getTitle = (content) => {
  *
  * @param content
  */
-const getPreview = (content: string) => {
+const getPreview = (content: string, searchQuery?: string) => {
   let preview = '';
   let lines = 0;
+
+  // contextual note previews
+  if (searchQuery?.trim()) {
+    const terms = getTerms(searchQuery);
+
+    // use only the first term of a multi-term query
+    const firstTerm = terms[0].toLocaleLowerCase();
+    const leadingChars = 30 - firstTerm.length;
+
+    const regExp = new RegExp(
+      /* eslint-disable */
```

Comment:
I can't get eslint to stop changing my indentation here :(

Reply:
indentation I think is governed by `prettier` not `eslint`, so you can try disabling the auto-formatting with [`// prettier-ignore`](https://prettier.io/docs/en/ignore.html)

```ts
// prettier-ignore
const regExp = new RegExp(
	…
);
```

Reply:
THANK YOU